### PR TITLE
✨ Added query option to `DuckDBToSQLServer`

### PR DESCRIPTION
**Pull request by angelika233**
## Summary
Added `duckdb_query` parameter to `DuckDBToSQLServer` flow

## Importance
User can specify query to filter the data e.g using WHERE clause

## Checklist

This PR:

- [x] adds new tests (if appropriate)
- [x] updates docstrings for any new functions or function arguments (if appropriate)
- [x] updates `CHANGELOG.md` with a summary of the changes

### CODE REVIEW

**Review by Trymzet — CHANGES_REQUESTED** [review]
Minor comments

**Comment by Trymzet** [comment]
Need to double check the schema exists test before moving on

**Review by Trymzet — CHANGES_REQUESTED** [review]
Just need to remove the prints

File: tests/integration/flows/test_duckdb_to_sql_server.py [hunk 1]
```diff
@@ -21,15 +23,28 @@ def duckdb():
...
+        secret=credentials_secret, vault_name=vault_name
+    )
+    yield json.loads(credentials_str)
+
+
 def test__check_if_schema_exists(duckdb):
-    duckdb.run(f"CREATE SCHEMA {SCHEMA}")
-    assert not duckdb._check_if_schema_exists(SCHEMA)
+    duckdb.run(f"CREATE SCHEMA {DUCKDB_SCHEMA}")
+    assert not duckdb._check_if_schema_exists(DUCKDB_SCHEMA)
```
[the remark this answers is not in the document]
Comment (Trymzet) [reply tests/integration/flows/test_duckdb_to_sql_server.py:41]: @angelika233 I forgot, was this resolved?

[the remark this answers is not in the document]
Comment (angelika233) [reply tests/integration/flows/test_duckdb_to_sql_server.py:41]: If I remember correctly @winiar93 fixed this in one of his PR

[the remark this answers is not in the document]
Comment (Trymzet) [reply tests/integration/flows/test_duckdb_to_sql_server.py:41]: I don't see it anywhere, can you remove this test and create an issue to fix it?

**Review by Trymzet — APPROVED** [review]
Looks good

**Pull request merged**